transposer/plugins/org.polarsys.kitalpha.transposer.rules.handler.fcore/model/mapping.fcore

```diff
@@ -12,8 +12,8 @@
           <type xmi:type="pattern:TypePatternSubstitution" xmi:id="_dBSO5gxWEeCHp6NSwmu3Pw"/>
         </orchestrationParameters>
       </orchestrationParameterContainer>
-      <invocations xmi:id="_lGr_4Mr3EeOVG-9pDQwlGA" name="mapping invocation" invokedActivity="#_dBSO5wxWEeCHp6NSwmu3Pw">
-        <invocationContractContainer xmi:id="_lGr_4cr3EeOVG-9pDQwlGA"/>
+      <invocations xmi:id="_gzBOYC_WEeu0bNPqDcB3TA" name="mapping invocation" invokedActivity="#_dBSO5wxWEeCHp6NSwmu3Pw">
+        <invocationContractContainer xmi:id="_gzBOYS_WEeu0bNPqDcB3TA"/>
```

Comment:
Can you please document all the changes done in this file in the PR/commit text ?